debian/changelog

```diff
@@ -1,3 +1,21 @@
+ros-melodic-realsense2-camera (92.2.13-0ubuntu) bionic; urgency=medium
```

Comment:
I've never seen this before, is this a good best practice?

Reply:
idk, it is probably better to rename the package to something like greenzie-ros-realsense2-camera when building the deb. 

But this likely works and is 'idiot proof' as opposed to me renaming the package in all the right spots in our code and this code :stuck_out_tongue_closed_eyes: